solutions/swb-reference/src/SWBStack.ts

```diff
@@ -115,7 +113,6 @@ export class SWBStack extends Stack {
       MAIN_ACCT_ALB_ARN_OUTPUT_KEY,
       SWB_DOMAIN_NAME_OUTPUT_KEY,
       MAIN_ACCT_ALB_LISTENER_ARN_OUTPUT_KEY,
-      ECR_REPOSITORY_NAME_OUTPUT_KEY,
       VPC_ID_OUTPUT_KEY,
       ALB_SUBNET_IDS,
```

Comment:
consider double checking if ALB and related infrastructure components are still needed.